### ISSUE TITLE
semi white transparent view on OS X 10.14.1

### ISSUE THREAD

**Issue by sgovindan** [skycocker/NameAndPassword#4]
After upgrading my mac to mojave 10.14.1. The SFAuthorizationView is being displayed with the semi white semi transparent square box around the user icon and text entry fields. Please refer the attached screenshots.

Issue screen snap on OSX 10.14.1
![white_blur_view_on_osx_10 14 x](https://user-images.githubusercontent.com/34024493/56154613-950fee00-5fd6-11e9-91ce-16eecba31103.jpeg)

Working snap on OSX 10.13.x 
![working_on_osx_10 12](https://user-images.githubusercontent.com/34024493/56154614-950fee00-5fd6-11e9-9128-4bdc48146a86.jpeg)

**Comment by amitt0488** [comment]
I am also facing this issue. @sgovindan @skycocker  Did you get the solution of this issue?

**Comment by skycocker** [comment]
@amitt0488 @sgovindan no, unfortunately I still didn't try to fix this problem so far
